include/wx/msw/webview_ie.h

```diff
@@ -35,6 +36,11 @@ class DocHostUIHandler;
 class wxFindPointers;
 class wxIInternetProtocol;
 
+#define IE_EMULATION_LEVEL 7000
```

Comment:
I think these constants shouldn't be defined in the header at all (but only in `webview_ie.cpp`), but if they should be here, they must have `wx` prefix.

As to why they shouldn't be here: AFAICS, they're only used by the test, but the test is not very useful as written. The function for setting the emulation level is supposed to make it possible for the scripts returning objects to work, so the test should check this instead of checking that some registry key was modified -- which, on its own, means nothing.